### PR TITLE
IPAllow: change top level key to "ip_allow".

### PR DESCRIPTION
**Pull request by SolidWallOfCode**

### CODE REVIEW

**Comment by randall** [comment]
[approve ci autest]

File: configs/ip_allow.yaml.default [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # That means you generally want to append your rules after the ones listed here.
```
Comment (randall) [inline configs/ip_allow.yaml.default]: ip_addr_acl referenced above this block

Comment (SolidWallOfCode) [reply configs/ip_allow.yaml.default]: Let me check, I thought I found all those references.

**Comment by randall** [comment]
./tests/gold_tests/autest-site/min_cfg/ip_allow.yaml still has the old key.

**Comment by SolidWallOfCode** [comment]
Gah.

**Comment by SolidWallOfCode** [comment]
[approve ci autest]

**Comment by randall** [comment]
[approve ci autest]

**Comment by zwoop** [comment]
[approve ci autest]

**Pull request merged**